src/ToolbarService.php

```diff
@@ -214,6 +215,15 @@ public function saveData(ServerRequest $request, ResponseInterface $response)
         ) {
             return null;
         }
+        // Skip plugins assets
+        if (strpos($path, '.') !== false &&
+            strpos($path, '/.') === false &&
+            strpos($path, '..') === false &&
+            $response->getStatusCode() === 200 &&
+            $this->checkFileExists($path)
```

Comment:
Instead looking for the file, could we skip all css, js and common image files? The checks would be simpler.

Reply:
first I wanted to not catch all files that not served by AssetMiddleware
https://github.com/cakephp/cakephp/blob/61a1ff9672430456a7a261f053911cab93793742/src/Routing/Middleware/AssetMiddleware.php#L86-L93
and then people should see other requests that are 404 not exists
just like the main assets behaviors in DebugKit 

but if `$response->getStatusCode() === 200 &&` is enough probably we could do this method simpler